schedule/yast/remote_ssh/remote_ssh_target_ftp_sle15.yaml

```diff
@@ -7,8 +7,8 @@ description: >
   any issue. Boots with ssh=1 parameter and waits for parallel job
   (remote_ssh_controller) to install the system.
 schedule:
-  - installation/isosize
-  - installation/bootloader
+  - '{{isosize}}'
```

Comment:
As you require conditional schedule, I would propose to completely remove that module from the schedule as we run it for all architecture in other test suites already.